odsApi_versioned_docs/version-7.3/whats-new/whats-new-in-this-release.md

```diff
@@ -19,28 +19,64 @@ Ed-Fi ODS / API Platform v7.3 and provides links to additional documentation.
 
 ### Data Model Changes
 
-Ed-Fi ODS / API v7.3 adds support for [Ed-Fi Data Standard
+Ed-Fi ODS / API v7.3 adds [Ed-Fi Data Standard
 v5.2](https://edfi.atlassian.net/wiki/spaces/EFDS5/overview) implementation, with
```

Comment:
`/reference/data-exchange/data-standard`

Reply:
DS space wasn't ready when I started working on this. I will update.